src/style.css

```diff
@@ -0,0 +1,25 @@
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: 400;
+  margin: 1em 0;
+}
+
+h1,
+p {
+  margin: 0;
+}
+
+a {
+  color: green;
+  font-size: 1em;

```

Comment:
do you really need this?

Reply:
Tried without these styles and it didn't match the size of the link in the mockup

Reply:
if you don't have `*` selector with font-size and you pushed and deployed all changes, it must work without this font-size